### ISSUE TITLE
Screen Reader Columns: approach doesn't work in screen reader browse (virtual cursor) mode

### ISSUE THREAD

**Issue by steve-haar** [vmware-clarity/starters#86]
From Amy Li

> Screen reader announcement is improved in focus mode but not in screen reader browse mode (aka virtual cursor), which still reads all the heading content when navigating grid cells and the core issue of having heading controls other than sorting is not fixed. ARIA datagrids only allow sorting controls in the headings anything beyond that will cause screen reader issues. Please see Santina's recommendations in this Jira ticket under Resolution Details. This is also related to the same [VPAT-2938](https://jira.eng.vmware.com/browse/VPAT-2938) issue mentioned above. Note that because it is out of spec it may fail other assistive technologies that aren't covered here and can cause unforeseen issues in future updates of ATs or browsers.

This approach may be fundamentally flawed
